### PR TITLE
rgw: add multi-instances support when deploying multisite

### PR DESCRIPTION
**Pull request by guits**
This commit adds the multi-instances when deploying rgw multisite

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>
Co-authored-by: Dimitri Savineau <dsavinea@redhat.com>

### CODE REVIEW

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by guits** [comment]
@alimaredia FYI

File: tests/functional/rgw-multisite/secondary/hosts [hunk 1]
```diff
@@ -5,4 +5,4 @@ mon0
 osd0
 
 [rgws]
-osd0 rgw_multisite=True rgw_zone=mars rgw_zonemaster=False rgw_zonesecondary=True rgw_zonegroup=solarsystem rgw_zonegroupmaster=True rgw_zone_user=zone.user rgw_zone_user_display_name="Zone User" rgw_realm=milkyway rgw_multisite_proto=http system_access_key=6kWkikvapSnHyE22P7nO system_secret_key=MGecsMrWtKZgngOHZdrd6d3JxGO5CPWgT2lcnpSt rgw_pull_proto=http rgw_pull_port=8080 rgw_pullhost=192.168.101.11 rgw_multisite_endpoint_addr=192.168.103.11
+osd0
```
Comment (guits) [inline tests/functional/rgw-multisite/secondary/hosts]: @dsavineau

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by dsavineau** [comment]
jenkins test centos-container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-non_container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-container-rgw_multisite

**Comment by guits** [comment]
jenkins test centos-non_container-rgw_multisite

**Pull request merged**